src/diffpy/labpdfproc/tools.py

```diff
@@ -28,6 +28,13 @@ def set_output_directory(args):
     return output_dir
 
 
+def _parse_file_list_file(input_path):
+    with open(input_path, "r") as f:
+        lines = [line.strip() for line in f]
```

Comment:
I would move the "Path" to this line and instead of line call it filepath or sthg like that, so it reads more like `file_paths = [Path(file_path.strip()) for file_path in f]` which is more intelligible of what the code is meant to be doing.  I wonder if it is more readable if we use readlines too?  though this is ok too.